src/pages/my_page/user_profile/ProfilePictureUploader.tsx

```diff
@@ -101,9 +105,23 @@ export const ProfilePictureUploader = ({ personId }: ProfilePictureUploaderProps
             <AddAPhotoIcon color="primary" />
             <input accept=".jpg, .jpeg" onChange={handleFileChange} hidden type="file" />
           </IconButton>
+
           <Typography>{t('my_page.my_profile.identity.upload_profile_photo')}</Typography>
+          <Typography align="center">{t('my_page.my_profile.identity.upload_profile_photo_info')}</Typography>
         </Box>
       )}
+      <ConfirmDialog
+        open={openConfirmDialog}
+        title={t('my_page.my_profile.remove_profile_picture')}
+        onAccept={async () => {
+          await mutateProfilePicture.mutateAsync('');
+          toggleConfirmDialog();
+        }}
+        onCancel={toggleConfirmDialog}
+        isLoading={mutateProfilePicture.isLoading}
+        dialogDataTestId="confirm-remove-orcid-connection-dialog">
```

Comment:
`confirm-remove-orcid-connection-dialog`? (husk å se gjennom egen PR også :) )

Reply:
Yeah, dette var slapt 😬

Reply:
kan bruke `dataTestIds.ts` også, så blir det enklere for eirik med testing